src/geometry_base/road_geometry.cc

```diff
@@ -63,5 +63,72 @@ const api::Junction* RoadGeometry::do_junction(int index) const { return junctio
 
 const api::BranchPoint* RoadGeometry::do_branch_point(int index) const { return branch_points_.at(index).get(); }
 
+/*
+void RoadGeometry::SpacialReorganization(const SpacialReorganization::Type& type) {
+  // Sample the surface using Lane::ToInertialPosition.
+  const auto lanes = this->ById().GetLanes();
+
+  std::deque<maliput::geometry_base::MaliputPoint> points;
+  for (const auto& lane : lanes) {
+    const auto lane_length = lane.second->length();
+    for (double s = 0; s <= lane_length; s += 0.1) {
+      const auto lane_bounds = lane.second->lane_bounds(s);
+      for (double r = lane_bounds.min(); r <= lane_bounds.max(); r += 0.1) {
+        const auto inertial_pos = lane.second->ToInertialPosition({s, r, h}).xyz(); //return h to 0.
+        const maliput::geometry_base::MaliputPoint point{{inertial_pos.x(), inertial_pos.y(), inertial_pos.z()},
+                                                         lane.first};
+        points.push_back(point);
+      }
+    }
+  }
+  switch (type) {
+    case SpacialReorganization::Type::kKDTree:
+      spacial_reorganization_ = std::make_unique<maliput::geometry_base::KDTreeReorganization>(this, std::move(points));
+      break;
+    default:
+      MALIPUT_THROW_MESSAGE("Unsupported spacial reorganization type");
+      break;
+  }
+}
+*/
+
+api::RoadPositionResult RoadGeometry::DoToRoadPosition(const api::InertialPosition& inertial_position,
+                                                       const std::optional<api::RoadPosition>& hint) const {
+  MALIPUT_THROW_UNLESS(strategy_ != nullptr);
```

Comment:
I don't think this is possible.

Reply:
Do you mean that it is impossible that `strategy_ == nullptr`? I believe that if there is an instance of a road geometry and the one that creates that instance does not call `RoadGeometry::InitializeStrategy()` then `strategy_` would be a `nullptr`.

Reply:
It doesn't make sense my comment now. Solving.